OWNERS_ALIASES

```diff
@@ -21,3 +22,4 @@ aliases:
   - rhuss
   - skonto
   - warrenvw
+  - rhuss
```

Comment:
You're already in there.

Reply:
gotcha, doppelt hält besser ;-P

removed ....